rotkehlchen/db/upgrades/v34_v35.py

```diff
@@ -37,17 +48,21 @@ def _create_new_tables(cursor: 'DBCursor') -> None:
     """)
 
 
-def _rename_assets_identifiers(cursor: 'DBCursor') -> None:
+def _rename_assets_identifiers(write_cursor: 'DBCursor') -> None:
     """Version 1.26 includes the migration for the global db and the references to assets
-    need to be updated also in this database"""
-    cursor.execute('SELECT identifier FROM assets')
+    need to be updated also in this database.
+    We need to update the rows instead of just deleting and inserting because the rows are
```

Comment:
What do you mean? The assets table of the user db is referenced in many tables. So if we replace the `BIFI` asset with the new `eip155...` we have to do it in all the tables. Since we have foreign keys with `ON UPDATE CASCADE` we need to do an update so the rows reference the correct entry of the assets table